nixos/doc/manual/configuration/profiles/hardened.xml

```diff
@@ -0,0 +1,22 @@
+
+<section xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xmlns:xi="http://www.w3.org/2001/XInclude"
+         version="5.0"
+         xml:id="sec-profile-hardened">
+ <title>Hardened</title>
+ <para>
+  A profile with most (vanilla) hardening options enabled by default,
+  potentially at the cost of features and performance.
+ </para>
+ <para>
+  This includes a hardened kernel, and limiting the system information
+  available to procesess via de <filename>/sys</filename> and
```

Comment:
“through the”?

Reply:
`processes`

Reply:
My spanish brain keeps trying to swap "th" with "d" when I'm tired :P. Both fixed in 3, 2, 1...